src/next.js/index.ts

```diff
@@ -24,15 +24,16 @@ export const handle = async (req: Request, res: Response) => {
   // dynamic for middleware.
   let nextApp = nextAppsLRU.get(key);
   if (!nextApp) {
+    // @ts-expect-error
     nextApp = next({
       dev: false,
       dir: process.cwd(),
       hostname: "0.0.0.0",
       port,
     });
-    nextAppsLRU.set(key, nextApp);
+    nextAppsLRU.set(key, nextApp!);
   }
-  await nextApp.prepare();
+  await nextApp!.prepare();
   const parsedUrl = parse(url, true);
-  nextApp.getRequestHandler()(req, res, parsedUrl);
+  nextApp!.getRequestHandler()(req, res, parsedUrl);
```

Comment:
Is this (and other non-test code) changes needed?

Reply:
This was apart of the fix @leoortizz was working on for (I believe) https://github.com/firebase/firebase-tools/issues/6188
But I can remove it from this PR!

Reply:
Let's keep test fixes minimal, ideally not touching production code.

Also Leo's changes might trigger linter errors if we decide to more strictly enforce it later. I would rather prefer type casting in another pull request.

Reply:
this change was a work in progress, it was not intended to be part of the PR, I was just testing the GitHub actions. @JulienMartel it doesn't relate to that issue, it's just a type *"*fix*"*.

Let's remove anything other than disabling the Nuxt 2 test in the GitHub actions from this PR.

@alexastrum the Next custom server imports aren't coming with a type, so that's why we already have type errors at the moment, but 100% agree that it should not be part of this PR.